operator/go.mod

```diff
@@ -3,13 +3,30 @@ module github.com/kyma-project/kyma-operator/operator
 go 1.17
 
 require (
-	github.com/go-logr/logr v1.2.0
+	github.com/go-logr/logr v1.2.3
+	github.com/khlifi411/kyma-listener v0.0.0-20220630064745-f01257cf7ce7
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
-	k8s.io/client-go v0.23.0
-	sigs.k8s.io/controller-runtime v0.11.0
+	github.com/onsi/gomega v1.18.1
+	go.uber.org/zap v1.19.1
+	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
+	k8s.io/api v0.24.2
```

Comment:
This for sure is not correct. All these dependencies HAVE to stick to v0.23.0 I would expect only the kyma-listener dependency to be upgraded. All other things will be upgraded in the go1.18 migration that is currently being prepared in another PR.